lib/rls/objects/search_results.rb

```diff
@@ -0,0 +1,49 @@
+module RLS
+  # Search results from RLS
+  class SearchResults
+    # @return [String] the search query
+    attr_reader :display_name
+
+    # @return [Integer] current page
+    attr_reader :page
+
+    # @return [Integer] amount of results in this query
+    attr_reader :results
+
+    # @return [Integer] total amount of players returned by the query
+    attr_reader :total_results
+
+    # @return [Integer]
+    attr_reader :max_results_per_page
+
+    # @return [Array<Player>] array of Player objects
```

Comment:
"array of Player objects that have been fetched so far", or similar, may be more fitting